### PR TITLE
Changes to the new display command

### PR DESCRIPTION
**Pull request by StrangeOne101**
# Changes
- Made the count of sub abilities not include passives and combos
- Optimized the amount of things in tab completion
- Changed "Metal (#)" to "Metal Abilities (#)"
- Changed "Combos (#)" to "ELEMENT Combos (#)". Same with passives
- Changed the bind format to use different characters
- Updated the help command to show the help for addon elements
- Removed the ProjectKorra website reference from `/b help <element>`
- Fixed reloading the config making new lines under the separator
- Fixed the chat formatting being unable to be changed once set for the bind format

### CODE REVIEW

**Comment by JustAHuman-xD** [comment]
Appending an empty string with no chat retention, smart! Wish I had thought of that 😅

**Comment by StrangeOne101** [comment]
> Appending an empty string with no chat retention, smart! Wish I had thought of that 😅

Took a bit to get it lol. I tried doing `builder.reset()` but that just cleared the formatting of the last component inserted. Had to do a lot of reading on how the builder worked to figure it out lol

File: src/com/projectkorra/projectkorra/command/HelpCommand.java [hunk 1]
```diff
@@ -153,23 +156,20 @@ public void execute(final CommandSender sender, final List<String> args) {
...
+			sender.sendMessage(Element.CHI.getColor() + this.elementHelp.get(Element.CHI).replaceAll("(?i)/b display chi", Element.CHI.getSubColor() + "/b display chi" + Element.CHI.getColor()));
 		} else if (Arrays.asList(Commands.avataraliases).contains(arg)) {
-			sender.sendMessage(Element.AVATAR.getColor() + this.avatar.replace("/b display Avatar", Element.AVATAR.getSubColor() + "/b display Avatar" + Element.AVATAR.getColor()));
-			sender.sendMessage(ChatColor.YELLOW + this.learnMore + ChatColor.DARK_AQUA + "http://projectkorra.com/");
+			sender.sendMessage(Element.AVATAR.getColor() + this.elementHelp.get(Element.AVATAR).replaceAll("(?i)/b display avatar", Element.AVATAR.getSubColor() + "/b display avatar" + Element.AVATAR.getColor()));
+		} else if (Element.getElement(arg) != null && elementHelp.containsKey(Element.getElement(arg))) {
+			Element element = Element.getElement(arg);
+			sender.sendMessage(element.getColor() + this.elementHelp.get(element).replaceAll("(?i)/b display " + element.getName().toLowerCase(), element.getSubColor() + "/b display " + element.getName().toLowerCase() + element.getColor()));
```
Comment (ChristopherWMM) [inline src/com/projectkorra/projectkorra/command/HelpCommand.java:172]: Do we really need to differentiate different elements now? It seems like we could just use the code in the last else if.

Comment (ChristopherWMM) [reply src/com/projectkorra/projectkorra/command/HelpCommand.java:172]: Ah no, we want to support the hardcoded core elemental aliases and custom elements don't have those. Nevermind 🤪 That being said, we may want to rethink that in the future.

Comment (StrangeOne101) [reply src/com/projectkorra/projectkorra/command/HelpCommand.java:172]: Yeah, I also removed it at first before realizing lol. We will indeed need to review in future

**Review by ChristopherWMM — APPROVED** [review]

**Review by PrideInt — APPROVED** [review]
looks good to me👍🏼

**Pull request merged**